docs/guide/install.md

```diff
@@ -37,3 +37,61 @@ $ npx slidev
 > ```bash
 > echo 'shamefully-flatten=true' >> .npmrc
 > ```
+
+## Install on Docker
+
+If you need a rapid way to run a presentation, you can use the prebuilt (or even build your own) [docker](https://hub.docker.com/r/stig124/slidev) image (maintained by [stig124](https://github.com/Stig124))
+
+##### *(this is not meant as a developper environnement, as it uses only npm-fetched binaries)*
+
+Pull the image :
+
+```bash
+$ docker pull stig124/slidev:latest
+```
+
+> [podman](https://podman.io/) users can pull the image using the complete URL
+>
+> ```bash
+> $ podman pull docker.io/stig124/slidev:latest
+> ```
+
+
+Go into a directory with a `slides.md` file and run it :
+
+```bash
+$ docker run --rm -v .:/root/slides -p 3000:3030 stig124/slidev:latest
+```
+
+> You can change the port that will face outside by changing the number left of the port declaration => `3000:3030`
+
+Open a browser on `localhost:3000` and you're ready to go, you can leave at any time using `Ctrl` + `C`
+
+> Please note that if you have a `node_modules` folder within the folder with the `slides.md` file, It will be **deleted** *(You can add `-e KEEP=1` next to `--rm` in the command to revert this behaviour)* as it may blocks the starting of the container
+
+### Build the docker image
+
+If you want to build the image by yourself, you can :
+
+Clone the repo
+
+```bash
+$ git clone https://github.com/slidevjs/slidev.git
+```
+
+> As the image downloads `slidev` from npm, you can also just download the `Dockerfile` and `entrypoint.sh` from the `Github` website
+
+Go to the folder where you have the `Dockerfile` and `entrypoint.sh` file
```

Comment:
Feels a bit unnecessary as we just have them in the root. could just have this tree commands

```bash
$ git clone https://github.com/slidevjs/slidev.git
$ cd slidev
$ docker build -t <tag> .
```